lib/student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, name, parent_permission, classroom)
```

Comment:
- [ ] `name` and `parent_permission` should be optional and have default values of "Unknown" and 'true'`
```def initialize(age, classroom, name = "Unknown", parent_permission = true)```